environments/environment-Windows.yml

```
@@ -18,5 +18,6 @@ dependencies:
       - flask-cors === 3.0.10
       - flask_restx == 1.1.0
       - neuroconv @ git+https://github.com/catalystneuro/neuroconv.git@main#neuroconv[full]
+      - dandi @ git+https://github.com/dandi/dandi-cli.git@master
```

```suggestion
      - dandi >= 0.57.0
```

DANDI just cut a new release lol